Library/Formula/ejabberd.rb

```diff
@@ -36,9 +27,26 @@ def install
             "--enable-pgsql",
             "--enable-mysql",
             "--enable-odbc",
-            "--enable-pam"]
+            "--enable-pam"
+          ]
 
+    system "autoupdate"
+    system "./autogen.sh"
     system "./configure", *args
+
+    # rebar tries to clone goldrush using a git://github.com/ url which is no longer offered by github.
+    # Work around by cloning it into the expected place using https.
+    system "git", "clone", "--branch", "0.1.8", "https://github.com/DeadZen/goldrush.git", "deps/goldrush"
+
+    # Before Snow Leopard, the pam header files were in /usr/include/pam instead of /usr/include/security.
+    # https://trac.macports.org/ticket/26127
+    if MacOS.version <= :leopard
```

Comment:
Comment mentions Snow Leopard

Reply:
Yeah, that comment in MacPorts is a little ambiguous to me. The way I ended up interpreting it is that `pam_appl.h` locations are:

- tiger: `/usr/include/pam`
- leopard: `/usr/include/pam`
- snow leopard: `/usr/include/security`

Reply:
My mistake, appologies.
I misread it as "< :leopard", while the comment says "before snow leopard".